catt/cli.py

```diff
@@ -73,19 +73,19 @@ def process_path(ctx, param, value):
     return path
 
 
-def get_device(ctx, param, value):
-    try:
-        return ctx.default_map["aliases"][value]
-    except KeyError:
+def process_device(ctx, param, value):
+    if is_ipaddress(value):
         return value
+    else:
+        return ctx.default_map["aliases"].get(value, value)
```

Comment:
Maybe the aliases should have higher priority? I can't see them conflicting with IP addresses, but just a thought.

Reply:
You're right, I just don't like checking ip-addresses against the alias list.

Reply:
That's fair.